File: data/src/main/java/com/github/gmetal/data/entity/RequestTokenEntity.java
package com.github.gmetal.data.entity;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
public class RequestTokenEntity {

    @JsonField
    public boolean success;

    @JsonField(name = "expires_at")
    public String expiresAt;

    @JsonField(name = "request_token")
    public String requestToken;
}

File: data/src/main/java/com/github/gmetal/data/entity/RequestTokenEntity.kt
package com.github.gmetal.data.entity

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class RequestTokenEntity {

    var success: Boolean = false

    @SerialName("expires_at")
    var expiresAt: String? = null

    @SerialName("request_token")
    var requestToken: String? = null
}
